assets/airports/loww.json

```diff
@@ -386,448 +362,339 @@
         "WW983": ["N48d06m38.15", "E16d42m40.10"],
         "WW985": ["N48d16m15.72", "E16d38m38.30"],
         "WW987": ["N48d27m47.10", "E16d33m46.60"],
-
         "WW988": ["N47d52m59.15", "E16d25m10.61"],
         "WW989": ["N47d58m41.50", "E16d55m35.34"],
         "WW991": ["N47d52m13.30", "E16d58m42.74"],
         "XANUT": ["N47d06m58.57", "E15d54m42.77"]
-
     },
     "runways":[
         {
-            "name":        ["11", "29"],
+            "name": ["11", "29"],
             "name_offset": [[0, 0], [0, 0]],
-            "end":    [["N48d7m22.13", "E16d32m00.15"], ["N48d6m32.57", "E16d34m32.3"]],
-            "length":      3.4923,
-            "ils":         [true, true]
+            "end": [
+                ["N48d7m22.13", "E16d32m00.15"],
+                ["N48d6m32.57", "E16d34m32.3"]
+            ],
+            "length": 3.4923,
+            "ils": [true, true]
         },
         {
-            "name":        ["16", "34"],
+            "name": ["16", "34"],
             "name_offset": [[0, 0], [0, 0]],
-            "end":    [["N48d7m11.21", "E16d34m41.46"], ["N48d5m19.07", "E16d35m28.87"]],
-            "length":      3.5888,
-            "ils":         [true, true]
+            "end": [
+                ["N48d7m11.21", "E16d34m41.46"],
+                ["N48d5m19.07", "E16d35m28.87"]
+            ],
+            "length": 3.5888,
+            "ils": [true, true]
         }
     ],
     "sids": {
```

Comment:
Understandable logic, but if an instrument procedure exists in reality, it must exist in the openscope airport file. We don't have to _use_ it for anything, but it will need to be defined.